app/models/measurement.rb

```diff
@@ -1,5 +1,7 @@
 class Measurement < ActiveRecord::Base
   belongs_to :person
+  scope :descending, -> { order(date: :desc) }
```

Comment:
Could we call this `ordered` instead, if it's the order in which they're meant to be presented?